api-reference/10 UI Components/CollectionWidget/1 Configuration/onSelectionChanged.md

```diff
@@ -5,7 +5,7 @@ default: null
 ---
 ---
 ##### shortDescription
-A function that is executed when a collection item is selected or selection is canceled.
+A function that is executed after selection changes.
```

Comment:
I would use the verb "call". Usually you would see a phrase "call a function", but I don't think "execute a function" is that common. 

```suggestion
A function that is called after selection changes.
```